data/json/monsters/mammal.json

```
@@ -162,6 +162,38 @@
     "special_attacks": [ [ "EAT_FOOD", 120 ] ],
     "flags": [ "SEES", "SMELLS", "HEARS", "WARM", "SWIMS", "ANIMAL", "STUMBLES", "PATH_AVOID_DANGER_1" ]
   },
+  {
+    "id": "mon_lab_rat",
+    "type": "MONSTER",
+    "name": { "str": "lab rat" },
+    "description": "The albino Norway rat, an omnivorous rodent with sleek white fur and a long, rough tail.  While its ancestors were harbingers of plague, these clever animals are omnipresent in modern biomedical research.",
+    "default_faction": "rat",
+    "bodytype": "pig",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "320 ml",
+    "weight": "320 g",
+    "hp": 6,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "r",
+    "color": "white",
+    "aggression": -15,
+    "morale": -50,
+    "melee_skill": 3,
+    "melee_dice": 1,
+    "melee_dice_sides": 1,
+    "melee_cut": 1,
+    "dodge": 2,
+    "vision_day": 30,
+    "vision_night": 15,
+    "harvest": "mammal_tiny",
+    "path_settings": { "max_dist": 10 },
+    "fear_triggers": [ "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "special_attacks": [ [ "EAT_FOOD", 120 ] ],
+    "flags": [ "SEES", "SMELLS", "HEARS", "WARM", "SWIMS", "ANIMAL", "STUMBLES", "PATH_AVOID_DANGER_1" ]
```

```suggestion
    "fear_triggers": [ "PLAYER_CLOSE", "SOUND", "FRIEND_DIED", "FIRE" ],
    "death_function": [ "NORMAL" ],
    "special_attacks": [ [ "EAT_FOOD", 120 ] ],
    "flags": [ "SEES", "SMELLS", "HEARS", "KEENNOSE", "WARM", "SWIMS", "ANIMAL", "PATH_AVOID_DANGER_1" ]
```
I wouldn't say they move that erratically on the size of a tile.

---

That makes a lot of sense to change, thank you for pointing it out. That was copied from the black rat's mammals.json entry; should they be updated similarly, or would that be outside the scope of this PR?